.github/workflows/theme-ci.yml

```diff
@@ -116,9 +115,13 @@ jobs:
         if: success()
         run: echo "BUILD_STATE=successful" >> $GITHUB_ENV
 
-      - name: Packaging
+      - name: Add files to zip
         run: |
-          yarn compress
+          mkdir -p ./tmp/default
```

Comment:
since we don't more use `yarn compress`, is that used somewhere else, or script can be deleted?

Reply:
will be removed latter